azext_iot/tests/digitaltwins/__init__.py

```diff
@@ -160,23 +160,22 @@ def tearDown(self):
 
     # Needed because the DT service will indicate provisioning is finished before it actually is.
     def wait_for_hostname(
-        self, instance: dict, wait_in_sec: int = 10, interval: int = 4
+        self, instance: dict, wait_in_sec: int = 10, interval: int = 7
     ):
         from time import sleep
-
-        while interval >= 1:
-            logger.info(
-                "Waiting :{} (sec) for provisioning to complete.".format(wait_in_sec)
+        sleep(wait_in_sec)
```

Comment:
Just curious - if we're adding the `dt wait` command to our tests - do we need to actually sleep first? In other words, does the `dt wait` command do an immediate poll, or does it wait first before polling? Not a huge deal, just always looking for a way to shave a few seconds off of the tests :)

Reply:
Sadly it polls right away so we can increase the number of polls or let the command sleep first. Since this helper function is used after functions that take a while to complete, it makes sense to sleep first